### ISSUE TITLE
Query - Resizing Crop Target

### ISSUE THREAD

**Issue by alpeshgit** [screen-share/element-capture#11]
What if the dimensions of the DOM element selected as crop target are updated?

Will it affect the recorded stream?

**Comment by eladalon1983** [comment]
Yes, as is currently the case when you capture a tab or a window and the user resizes them.

**Comment by eladalon1983** [comment]
@alpeshgit, shall we close this issue?
